src/storage/src/encoding/mod.rs

```diff
@@ -90,8 +90,8 @@ pub trait Decoding {
 
 /// `RowSerde` provides the ability to convert between Row and KV entry.
 pub trait RowSerde: Send + Sync + Clone {
-    type Serializer: Encoding;
-    type Deserializer: Decoding;
+    type Serializer: Encoding + Send;
+    type Deserializer: Decoding + Send;
```

Comment:
Suggest naming:

```suggestion
    type Serializer: RowSerialize;
    type Deserializer: RowDeserialize;
```

Also rename `mod encoding` to `mod row_serde`. 🥵